iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Devices.java

```diff
@@ -164,19 +164,21 @@ public DeviceTwinName getDeviceTwinNames()
 
                             final Map<String, TwinServiceModel> twinsMap = twins.getLeft();
                             final String responseContinuationToken = twins.getRight();
+
+                            devices = devices.stream().filter(device -> twinsMap.containsKey(device
+                                    .getDeviceId()))
+                                    .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Do we follow any conventions for lambda expression? something of this sort.
devices.stream().filter(device -> twinsMap.containsKey(device.getDeviceId()))
                          .collect(Collectors.toCollection(ArrayList::new));

This is not a blocker for me, but Isaac asked about this in my previous PR.

Reply:
That looks a little cleaner. I'll reformat.